content/blog/monica-granados.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Open Science for the Climate Movement, a chat with Dr Monica Granados"
+author: Monica Granados, Julie Lowndes, Stefanie Butland
+date: '2022-12-20'
+slug: monica-granados
+categories:
+  - community
+tags:
+  - community-call
+banner: img/blog/monica-granados-square.png
+---
+
+## A casual conversation with Dr. Monica Granados to discuss the Open Climate Campaign and how we can reimagine open science as part of the climate movement
```

Comment:
Suggest the header be instead: 

"Reimagining open science as part of the climate movement"